CHANGES_NEXT_RELEASE

```diff
@@ -12,3 +12,5 @@
 [cygnus-ngsi][Elasticsearch] Fix a bug that non-Latin characters such as Japanese are garbled. (#1839)
 [cygnus-ngsi][MongoSink]Fix on mongo collection name asignation. (#1845)
 [cygnus-ngsi][MongoSink]Fix on mongo recvTime Date format.
+[cygnus-common][MongoBackend] Fix persistence error mongo authentication and side effect. (#1849)
+[cygnus-ngsi][MongoSink] Add a new mongoAuthSource parameter to create the Mongo backend.
```

Comment:
```suggestion
[cygnus-ngsi][MongoSink][MongoSTH] Add a new mongoAuthSource parameter to create the Mongo backend.
```

Given that, as far as I understand, both sinks are "solved" by this PR.

Reply:
Yes, theoretically this PR solves both sinks